packages/components/organisms/f-checkout/src/components/AgeVerification.vue

```diff
@@ -101,7 +101,10 @@ export default {
         ...mapState(VUEX_CHECKOUT_MODULE, ['customer']),
 
         days () {
-            return Array.from({ length: 31 }, (_, index) => {
+            const month = Number(this.selectedDate.month) + 1;
```

Comment:
Personally I would have this as an imported method from a service/helper file, and then you can test that separately, this should help keep logic out of the component where possible.